migration/idempotent/001-base.sql

```diff
@@ -2554,218 +2554,149 @@ LANGUAGE PLPGSQL;
-                FROM _timescaledb_internal.hypertable_chunk_local_size ch
-                (
-                    hypertable_schema,
-                    hypertable_name,
-                    hypertable_id,
-                    chunk_id,
-                    chunk_schema,
-                    chunk_name,
-                    total_bytes,
-                    index_bytes,
-                    toast_bytes,
-                    compressed_total_size,
-                    compressed_index_size,
-                    compressed_toast_size,
-                    compressed_heap_size
-                )
-                WHERE ch.hypertable_schema = schema_name_in
-                GROUP BY ch.hypertable_name, ch.hypertable_schema;
-            END IF;
-        END;
-        $function$
-        LANGUAGE plpgsql STRICT STABLE;
-        REVOKE ALL ON FUNCTION _prom_catalog.hypertable_local_size(text) FROM PUBLIC;
-        GRANT EXECUTE ON FUNCTION _prom_catalog.hypertable_local_size(text) to prom_reader;
-
-        CREATE OR REPLACE FUNCTION _prom_catalog.hypertable_node_up(schema_name_in text)
-        RETURNS TABLE(hypertable_name text, node_name text, node_up boolean)
-        SECURITY DEFINER
-        SET search_path = pg_catalog, pg_temp
-        AS $function$
-            -- list of distributed hypertables and whether or not the associated data node is up
-            -- only ping each distinct data node once and no more
-            -- there is no guarantee that a node will stay "up" for the duration of a transaction
-            -- but we don't want to pay the penalty of asking more than once, so we mark this
-            -- function as stable to allow the results to be cached
-            WITH dht AS MATERIALIZED (
-                -- list of distributed hypertables
-                SELECT
-                    ht.table_name,
-                    s.node_name
-                FROM _timescaledb_catalog.hypertable ht
-                JOIN _timescaledb_catalog.hypertable_data_node s ON (
-                    ht.replication_factor > 0 AND s.hypertable_id = ht.id
-                )
-                WHERE ht.schema_name = schema_name_in
-            ),
-            up AS MATERIALIZED (
-                -- list of nodes we care about and whether they are up
-                SELECT
-                    x.node_name,
-                    _timescaledb_internal.ping_data_node(x.node_name) AS node_up
-                FROM (
-                    SELECT DISTINCT dht.node_name -- only ping each node once
-                    FROM dht
-                ) x
-            )
-            SELECT
-                dht.table_name::text as hypertable_name,
-                dht.node_name::text as node_name,
-                up.node_up
+IF NOT _prom_catalog.is_timescaledb_installed() THEN
```

Comment:
I know this is being nitpicky, but is there a reason that the contents of the BEGIN are not nested?

Reply:
Yeah, I'm 50/50 on it. When I'm looking at an individual function, the extra nest was distracting. But I don't feel strongly either way. WDYT? @JamesGuthrie

Reply:
I would prefer being able to easily see where the `BEGIN/END` block begins and ends, so I would be for indenting the contents one level.

Reply:
okay, i'll indent, but I will also wrap each function individually so you don't have to scroll a ton to figure out why it's indented